tools/RAiDER/checkArgs.py

```diff
@@ -58,7 +59,7 @@ def checkArgs(args):
                         d.strftime('%Y%m%dT%H%M%S'),
                     )
                 )
-                hydroFilename = None # only the 'wetFilename' is used for the station_file
+                hydroFilename = '' # only the 'wetFilename' is used for the station_file
```

Comment:
Can you confirm/ justify the change?

Reply:
If a user asks for delays a point (s), there is no [hydroFilename (f)](https://github.com/dbekaert/RAiDER/blob/74535ed4baac1d960ec7f6f3431f9a202d16e6e0/tools/RAiDER/cli/raider.py#L222-L225). 

Then, if hydroFilename/f is None, the string replacement will fail for delays on slant  (L324, L326). hydroFilename/f isn't used in the subsequent call to writeDelays for this AOI type.

Reply:
Is there an issue ticket? This should be a separate PR.

Reply:
@bbuzz31 this is not a big deal to me but I do tend to agree on a separate PR for this one. We could do some more thorough testing on filenames for various input options.